tsconfig.json

```diff
@@ -3,6 +3,7 @@
   "compilerOptions": {
     "declaration": true,
     "sourceMap": true,
+    "lib": ["es2016"],
```

Comment:
Hi @iffa! Thanks for contributing this PR, it LGTM. Could you explain why this change above is needed? I don't see any ES2016-related changes in the rest of the PR.

Reply:
IIRC this was only done because WebStorm was not particularly happy about the code using ES2016 functionality (i.e. Array.prototype.includes) without explicitly specifying that it is used